pkg/source/hdfsprotocol/hdfs_source_client.go

```diff
@@ -61,14 +61,14 @@ type hdfsSourceClient struct {
 
 // hdfsFileReaderClose is a combination object of the  io.LimitedReader and io.Closer
 type hdfsFileReaderClose struct {
-	limited io.Reader
-	c       io.Closer
+	reader io.Reader
```

Comment:
I don't think rename limited to reader is a good idea

Reply:
How about `limited -> limitedReader` ?